velox/vector/BaseVector.h

```diff
@@ -584,10 +584,10 @@ class BaseVector {
     } else {
       *buffer = AlignedBuffer::allocate<T>(size, pool);
     }
+    (*buffer)->setSize(minBytes);
```

Comment:
Is this a bug fix? Would you explain what was the problem before the change?

Reply:
No. Might be better to get raw pointer after fully setup buffer? I can also revert it since it is unrelated to this change.

Reply:
Thank you for explaining. Might be better to revert as it is unrelated.